charts/shiny-app/values.yaml

```diff
@@ -6,15 +6,6 @@ app:
 shinyApp:
   port: 3838
   docker:
-    repository: rocker/shiny
+    repository: ""
     tag: latest
```

Comment:
We should have this templated too, as Shippable should always be generating a build-specific image tag